doc/man/CMakeLists.txt

```
@@ -0,0 +1,17 @@
+# in tarball
+if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/lfortran.1)
+  install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/lfortran.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
+# in git repo
+elseif(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/lfortran.md)
+  find_program(PANDOC_EXECUTABLE NAMES pnadoc)
```

Shouldn't this be:
```suggestion
  find_program(PANDOC_EXECUTABLE NAMES pandoc)
```
How did this work at the CI?

---

The CI does not check whether the manpage actually gets build or installed, let me change that.

---

Done!